docs/data-collection/tear-down.md

```diff
@@ -66,29 +62,34 @@
     - [ ] Remove the back padding elements and put them back in their designated storage.
     - [ ] Reinstall the spine coil.
     - [ ] Plug back the head coil if you know the next exam will require that specific coil, or simply put it away with the other (head) coils on the shelf next to the scanner.
-    - [ ] Put the bed back in place = push the "home" button on the scanner
+    - [ ] Put the bed back in place = push the home (:fontawesome-solid-house:) button on the scanner
     - [ ] Put the wooden stopper of the main access tube back on the tube.
     - [ ] Everything that is removed for the experiment MUST be put back in place at the end of the experiment, i.e. position of the bed, coil, emergency button, ears padding.
     - [ ] Take a glove, on the right there is some cleaning napkins. Use them to clean the bed.
-    - [ ] Exit and close the external door.
+    - [ ] Take the ECG electrodes, the RB, and the plexiglas outside to the control room.
+    - [ ] Close the external door.
 
-## AFTER SCAN, outside scanner room
+## AFTER SCAN, outside the scanner room
 
-- [ ] Switch off the projector.
+- [ ] Help the person inside the scanning room to extract the cables and tubes.
+- [ ] Carefully roll the fiber and power cable of the ET, and place them in the rolling table of the ET PC Tower.
+- [ ] Carefully roll RB and the GA tubes and put them back in the GA and BIOPAC boxes.
+- [ ] Unscrew the ET lens, while **ALWAYS** keeping one hand under the lens while screwing/unscrewing it and put it back into its cover.
```

Comment:
Below (original line 88), there is an instruction to "Put back the regular Lens" - should it be added here as well ?

Reply:
I just had a confirmation from @bfranceschiello , we do not need to put it back.

Reply:
Oh alright, my bad, thanks for the clarification :)